paperless.conf.example

```diff
@@ -27,8 +27,10 @@
 # Security and hosting
 
 #PAPERLESS_SECRET_KEY=change-me
-#PAPERLESS_ALLOWED_HOSTS=example.com,www.example.com
-#PAPERLESS_CORS_ALLOWED_HOSTS=http://example.com,http://localhost:8000
+#PAPERLESS_URL=https://example.com
```

Comment:
I would let the variables be empty, e.g.
`PAPERLESS_URL=` 
since the conf.example should show what the default values are. (Even though this wasn't the case with `PAPERLESS_ALLOWED_HOSTS` and CORS, I think it should be. People should be able to copy the `paperless.conf.example` file to `paperless.conf` without any changes to the default settings)

Reply:
Hmm, Im not sure about this, doesnt it seem confusing to include those options but leave them blank? The thing about the `.example` conf for me is that if someone copes the `.example` then all options are commented out, meaning copying it and running would be fine. The user need only un-comment what they want, in which case I think its helpful for there to be some indication of whats supposed to be there. Like `PAPERLESS_SECRET_KEY=change-me`?

Reply:
yes, that makes sense. And we already document the default in our `configuration.rst`. Then lets just keep the example.com domain here.